unicode_smilies/unicode_smilies.php

```diff
@@ -1,260 +1,2042 @@
+Smilies::add($b, ':pig', '&#x1F416;');
 
 // Baby Smileys
 
-	Smilies::add($b, ':baby', '&#x1F476;');
-//	Smilies::add($b, ':babycot', '&#x;');
-//	Smilies::add($b, ':pregnant', '&#x1F930;'); // Bad client support
-//	Smilies::add($b, ':stork', '&#x;');
+Smilies::add($b, ':baby', '&#x1F476;');
+//Smilies::add($b, ':babycot', '&#x;');
+//Smilies::add($b, ':pregnant', '&#x1F930;'); // Bad client support
+//Smilies::add($b, ':stork', '&#x;');
 
 // Confused Smileys
 
-	Smilies::add($b, ':confused', '&#x1F615;');
-	Smilies::add($b, ':shrug', '&#x1F937;');
-//	Smilies::add($b, ':stupid', '&#x;');
-//	Smilies::add($b, ':dazed', '&#x;');
+Smilies::add($b, ':confused', '&#x1F615;');
+Smilies::add($b, ':shrug', '&#x1F937;');
+//Smilies::add($b, ':stupid', '&#x;');
+//Smilies::add($b, ':dazed', '&#x;');
 
 // Cool Smileys
 
-//	Smilies::add($b, ':affro', '&#x;');
+//Smilies::add($b, ':affro', '&#x;');
 
 // Devil/Angel Smileys
 
-	Smilies::add($b, ':angel', '&#x1F47C;');
-	Smilies::add($b, ':cherub', '&#x1F47C;');
-//	Smilies::add($b, ':devilangel', '&#x;');
-//	Smilies::add($b, ':catdevil', '&#x;');
-//	Smilies::add($b, ':devillish', '&#x;');
-//	Smilies::add($b, ':daseesaw', '&#x;');
-//	Smilies::add($b, ':turnevil', '&#x;');
-//	Smilies::add($b, ':saint', '&#x;');
-//	Smilies::add($b, ':graveside', '&#x;');
+Smilies::add($b, ':angel', '&#x1F47C;');
+Smilies::add($b, ':cherub', '&#x1F47C;');
+//Smilies::add($b, ':devilangel', '&#x;');
+//Smilies::add($b, ':catdevil', '&#x;');
+//Smilies::add($b, ':devillish', '&#x;');
+//Smilies::add($b, ':daseesaw', '&#x;');
+//Smilies::add($b, ':turnevil', '&#x;');
+//Smilies::add($b, ':saint', '&#x;');
+//Smilies::add($b, ':graveside', '&#x;');
 
 // Unpleasent smileys.
 
-	Smilies::add($b, ':toilet', '&#x1F6BD;');
-//	Smilies::add($b, ':fartinbed', '&#x;');
-//	Smilies::add($b, ':fartblush', '&#x;');
+Smilies::add($b, ':toilet', '&#x1F6BD;');
+//Smilies::add($b, ':fartinbed', '&#x;');
+//Smilies::add($b, ':fartblush', '&#x;');
 
 // Sad smileys
 
-	Smilies::add($b, ':crying', '&#x1F622;');
-//	Smilies::add($b, ':prisoner', '&#x;');
-//	Smilies::add($b, ':sigh', '&#x;');
-
-// Smoking - only one smiley in here, maybe it needs moving elsewhere?
-
-	Smilies::add($b, ':smoking', '&#x1F6AC;');
+Smilies::add($b, ':crying', '&#x1F622;');
+//Smilies::add($b, ':prisoner', '&#x;');
+//Smilies::add($b, ':sigh', '&#x;');
 
 // Sport smileys
 
-	Smilies::add($b, ':basketball', '&#x1F3C0;');
-	Smilies::add($b, '~bowling', '&#x1F3B3;');
-	Smilies::add($b, ':cycling', '&#x1F6B4;');
-	Smilies::add($b, ':darts', '&#x1F3AF;');
-	Smilies::add($b, ':fencing', '&#x1F93A;');
-	Smilies::add($b, ':juggling', '&#x1F939;');
-//	Smilies::add($b, ':skipping', '&#x;');
-//	Smilies::add($b, ':archery', '&#x;');
-	Smilies::add($b, ':surfing', '&#x1F3C4;');
-	Smilies::add($b, ':snooker', '&#x1F3B1;');
-	Smilies::add($b, ':horseriding', '&#x1F3C7;');
+Smilies::add($b, ':basketball', '&#x1F3C0;');
+Smilies::add($b, '~bowling', '&#x1F3B3;');
```

Comment:
This should be `:bowling` and not `-bowling` I think.

Reply:
modified